README.md

```diff
@@ -88,116 +59,108 @@ Key features:
 * Enchanting support
 * Support for adding and customizing materials, patterns and gems entirely trough config files
```

Comment:
Patterns was the old name for schematics, so I think we should replace the use of pattern with schematics